resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.spec.erb

```diff
@@ -93,6 +111,11 @@ Requires:         chkconfig
 Requires:         aaa_base
 %endif
 
+<% if !EZBake::Config[:is_pe_build] -%>
+# Required to get trustanchors for java
+BuildRequires: %{_certs_package}
```

Comment:
Sorry, I meant to ask about PE, but I assume that requirement is already satisfied with our dependency on puppet-agent. Sorry, that took my brain a while to parse